.github/workflows/testing.yml

```diff
@@ -5,6 +5,11 @@ on: push
 jobs:
   run-test-code:
     runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/bquan0/project-name:latest
```

Comment:
This should always use the most recent `CNERG` version since some users may not have generated their own image